commit a02dd8ddc7dde30ba71e87ab5e90bf910b68da2b
Author: Alexander Larsson <alexl@redhat.com>
Date:   Fri Nov 11 16:15:31 2011 +0100

    win32: Reset clip in _cairo_win32_surface_release_dest_image
    
    Without this we were using leftover clip regions from e.g. show_glyphs
    which made the fallback drawing results disappear.

diff --git a/src/cairo-win32-surface.c b/src/cairo-win32-surface.c
index 660aaba..59305f1 100644
--- a/src/cairo-win32-surface.c
+++ b/src/cairo-win32-surface.c
@@ -630,6 +630,8 @@ _cairo_win32_surface_release_dest_image (void                    *abstract_surfa
     if (!local)
 	return;
 
+    _cairo_win32_surface_set_clip_region (surface, NULL);
+
     if (!BitBlt (surface->dc,
 		 image_rect->x, image_rect->y,
 		 image_rect->width, image_rect->height,
